### ISSUE TITLE
Prediction of user's reaction on the topic posted on Pantip.com

### ISSUE THREAD

**Issue by tao-pr** [bkkhack/hacknights#31]
Each of topics posted on Pantip.com gains some votes and gets either positive or negative reactions by readers. The information is ready on the web so it should be fun to hack on sentiment analysis.

I have made a quick repo which is able to download the topics, formulate them as training set and ready for analysis @ https://github.com/starcolon/pantip-libr

To save setup time, have the following prepreq installed:
- Python 3.4+
- Ruby 2.1+
- Scikit-learn 0.17+
- CouchDB
- RabbitMQ

**Comment by tao-pr** [comment]
#progress

60~70%  of "training accuracy" so far. _BUT_ the trained models are too huge (one of them is bigger than 4GB) and the training time is painful. Next time I'll pick it for improvements.

Accuracy so far
https://github.com/starcolon/pantip-libr#how-it-got-so-far

**Comment by ches** [comment]
Love the project idea :laughing:  Looks like a good start!

**Comment by tao-pr** [comment]
If a system can anticipate how popular / reactions of readers a post will get before it gets posted, it would help people who love writing reviews. haha. :sweat_smile: That's the idea behind this.
